docs/config.md

```diff
@@ -1822,36 +1833,99 @@ Default: `"5m"`
 
 Type: `duration`
 
-## `Provider`
-
-Configuration applied to the initial one-time announcement of fresh CIDs
-created with `ipfs add`, `ipfs files`, `ipfs dag import`, `ipfs block|dag put`
-commands.
+## `Provide`
 
-For periodical DHT reprovide settings, see [`Reprovide.*`](#reprovider).
+Configures CID announcements to the routing system, including both immediate 
```

Comment:
It's worth noting here that even though this is designed to apply to all routing systems, it only supports providing to the Amino DHT at this point (rather than theoretically multiple routing systems)